index.html

```diff
@@ -1,60 +1,70 @@
+  <title>To-Do-Manager</title>
 
-  <body>
-    <div class="header-section">
-      <section class="greeting">
-        <h2 class="title">
-          Hey, <input type="text" id="name" placeholder="Name Here!!" />
-        </h2>
-      </section>
-      
-      <header>
-        <h1>Task List ↴ </h1>
-        <form id="new-task-form">
-          <input type="text" name="new-task-input" id="new-task-input" placeholder="What would you like to add!!" />
-          <input type="submit" id="new-task-submit" value="Add Task" />
-          </form>
-          </header>
-    </div>
+  <link rel="stylesheet" href="./assets/main.css" />
+  <link rel="stylesheet" href="./assets/name/storeName.css" />
+  <link rel="stylesheet" href="./assets/custom-alert/alert.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" />
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+</head>
 
+<body>
+  <div class="header-section">
+    <section class="greeting">
+      <h2 class="title">
+        Hey, <input type="text" id="name" placeholder="Name Here!!" />
+      </h2>
+    </section>
 
-    <div class="alertt hide" id="custAlert3">
-      <span class="fas fa fa-check-circle" style="font-size:24px"></span>
-      <span class="msgt">Success: Task added Successfully</span>
-      <div class="close-btnt" id="alertClose">
-        <span class="fas fa-times"></span>
-      </div>
+    <header>
+      <h1>Task List ↴ </h1>
+      <form id="new-task-form">
+        <input type="text" name="new-task-input" id="new-task-input" placeholder="What would you like to add!!" />
+        <input type="submit" id="new-task-submit" value="Add Task" />
+      </form>
+    </header>
+  </div>
+
+
+  <div class="alertt hide" id="custAlert3">
+    <span class="fas fa fa-check-circle" style="font-size:24px"></span>
+    <span class="msgt">Success: Task added Successfully</span>
+    <div class="close-btnt" id="alertClose">
+      <span class="fas fa-times"></span>
     </div>
-    <div class="alertd hide" id="custAlert4">
-      <span class='fas fa-trash-alt' style='font-size:24px;'></span>
-      <span class="msgd">Success: Task deleted Successfully</span>
-      <div class="close-btnd" id="alertClose">
-        <span class="fas fa-times"></span>
-      </div>
+  </div>
+  <div class="alertd hide" id="custAlert4">
+    <span class='fas fa-trash-alt' style='font-size:24px;'></span>
+    <span class="msgd">Success: Task deleted Successfully</span>
+    <div class="close-btnd" id="alertClose">
+      <span class="fas fa-times"></span>
+    </div>
+  </div>
+
+  <div class="update hide" id="custAlert5" style="background-color: rgb(83, 83, 231);">
+    <span class='fa fa-edit' style='font-size:24px; color: rgb(2, 2, 59)'></span>
+    <span class="msgt">Success: Task Updated Successfully</span>
+    <div class="close-btnt" id="alertClose">
+      <span class="fas fa-times"></span>
```

Comment:
colors should be updated via alert.css file
In css file you kept yellow color as it is and simply trying to overwrite it by inline css

Also you have given wrong bg color
use `#007bff` and respective 

In comment https://github.com/malivinayak/To-Do-Manager/pull/45#pullrequestreview-1658035037 I have clearly mention about darker shade at cross sign
> Also at cross sign there must be darker shade of color

You overlooked that too

> In css file you kept yellow color as it is and simply trying to overwrite it by inline css

**open-source is work of choice
don't rush and add dead + unnecessary code**